OpenShift/helm/nexus-repository-manager/Chart.yaml

```diff
@@ -1,8 +1,25 @@
-apiVersion: v1
-name: sonatype-nexus
-version: 0.2.0
-appVersion: 3.20.1-01
-description: Sonatype Nexus is an open source repository manager
+apiVersion: v2
```

Comment:
continuing my thoughts from the iq chart: what *would* happen if we targeted helm 2 to be more compatible with current operators?

Reply:
we should clarify if this is truly the case. Helm 2 is a security hole in k8s so I thought most folks moved to 3 to get rid of Tiller. I think the operators work with 2 or 3 but they are holding out hope that 3 might be able to go past a level 2 operator (install and update) all the way to full lifecycle management.

Reply:
I found a blog from last June with this:

A note on Helm 2 vs Helm 3
This post covers the **Operator SDK v0.8.1, which relies on Helm 2** code in order to process your chart into an Operator. Future versions of the SDK should also be compatible with Helm 3 when it becomes stable. Your usage of the SDK should not have to change and these modifications should be under-the-hood. Stay tuned for more details about this as the Helm 3 effort matures.

Reply:
I should add that 0.17.1 is the latest release of the SDK

Reply:
Seems like 0.14.0 upgraded to helm3, we should check with Yanai
https://github.com/operator-framework/operator-sdk/releases/tag/v0.14.0